agent/command/s3_put.go

```diff
@@ -379,11 +379,18 @@ retryLoop:
 				}
 				filesList, err = b.Build()
 				if err != nil {
-					return errors.Wrapf(err, "processing local files include filter '%s'",
-						strings.Join(s3pc.LocalFilesIncludeFilter, " "))
+					// Should replicate default behavior of not erroring when filter doesn't match any files
```

Comment:
I'm having trouble parsing this comment 😅 "not erroring when filter doesn't match any files when we error while looking through deleted directories" doesn't seem like a sentence. Can we reword? It sounds like you want to say something like, by default local files include filter should treat files as optional, so we skip erroring in this case.